src/Avalonia.Controls/Notifications/Notification.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Linq;
+using System.Reactive.Linq;
+using Avalonia.Controls.Primitives;
+using Avalonia.Interactivity;
+using Avalonia.LogicalTree;
+
+namespace Avalonia.Controls.Notifications
+{
+    public class Notification : ContentControl
```

Comment:
Feels a bit weird to have a control called `Notification` and an interface called `INotification` and them not be related.

Reply:
renamed to notificationcard and the simplenotification is now just notification which relates to the interface.